lib/vmdb/appliance.rb

```diff
@@ -14,6 +14,11 @@ def self.CODENAME
       "Ivanchuk".freeze
     end
 
+    def self.RELEASE
+      release_file = File.join(File.expand_path(Rails.root), "RELEASE")
```

Comment:
Minor, but since Rails.root is a Pathname this can be `Rails.root.join("RELEASE")`